warehouse/query-core/src/test/java/datawave/query/jexl/visitors/RewriteNullFunctionsVisitorTest.java

```diff
@@ -0,0 +1,58 @@
+package datawave.query.jexl.visitors;
+
+import datawave.query.exceptions.InvalidQueryTreeException;
+import datawave.query.jexl.JexlASTHelper;
+import datawave.query.jexl.visitors.validate.ASTValidator;
+import datawave.test.JexlNodeAssert;
+import org.apache.commons.jexl2.parser.ASTJexlScript;
+import org.apache.commons.jexl2.parser.ParseException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RewriteNullFunctionsVisitorTest {
+    
+    @Test
+    public void testIsNotNullWithSingleField() throws ParseException {
```

Comment:
Good start on test coverage, but I would like to see some for `filter:isNull` functions as well.